### DISCUSSION TITLE
a/blog-images-with-cloudflare-r2

### DISCUSSION THREAD

**Discussion by giscus** [sincerefly/giscus-data#16] [Announcements]
# a/blog-images-with-cloudflare-r2

https://blog.yasking.org/a/blog-images-with-cloudflare-r2.html

**Comment by zeruns** [comment]
按照步奏操作，访问报错Error 1101

**Reply by sincerefly** [reply]
不应该啊 🤔 Error 1101 应该是 Worker 抛出的错误，要不你对照本文的参考链接：https://www.eallion.com/cdn-cname-cloudflare/ 进一步核对下
